### ISSUE TITLE
[BUG] Darkmode not working on Home Page

### ISSUE THREAD

**Issue by KendallDoesCoding** [pushkaraj2007/Rockout#13]
The darkmode toggle doesn't work on the home page.

**Comment by pushkaraj2007** [comment]
Thanks for reporting the issue, I'm going to fix it soon...

**Comment by pushkaraj2007** [comment]
This issue has been resolved!

**Comment by KendallDoesCoding** [comment]
@pushkaraj2007 - now it looks bad in my opinion, I prefered the orange background!!

**Comment by pushkaraj2007** [comment]
@KendallDoesCoding 

I think so too, do you have any better idea to add dark mode? If not, would you like to create a PR to switch to the previous orange background?

**Comment by KendallDoesCoding** [comment]
> @KendallDoesCoding
> 
> I think so too, do you have any better idea to add dark mode? If not, would you like to create a PR to switch to the previous orange background?

I mean, it doesn't have to be darkmode, we can just toggle and change the background color, orange and maybe red may look good?

**Comment by KendallDoesCoding** [comment]
@pushkaraj2007 - what do you think?

**Comment by pushkaraj2007** [comment]
@KendallDoesCoding 

Yeah, I think so maybe just keeping the same background all time.
Would you like to create a PR with your Ideas about this?
